pptx_blueprint/__init__.py

```diff
@@ -33,14 +33,47 @@ def replace_text(self, label: str, new_text: str) -> None:
         for shape in shapes:
             shape.text = new_text
 
-    def replace_picture(self, label: str, filename: _Pathlike) -> None:
+    def replace_picture(self, label: str, filename: _Pathlike, *, do_not_scale_up: bool = False) -> None:
         """Replaces rectangle placeholders on one or many slides.
 
         Args:
             label (str): label of the placeholder (without curly braces)
             filename (path-like): path to an image file
+            do_not_scale_up (bool): deactivates that the image is enlarged (default: False)
         """
```

Comment:
A default kwarg `scale_up=True` would be easier to understand

Reply:
I have replaced `do_not_scale_up` by ``scale_up` and changed how it's used.
Additionally, I added some more information to the doc string of `replace_image()`.